docs/guidelines/code-guidelines.md

```diff
@@ -8,6 +8,20 @@ This documents idiomatic conventions in the Go code that we follow for gaia deve
 2. [Go Common Mistakes](https://github.com/golang/go/wiki/CommonMistakes)
 3. [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
 
+## Maintainability
+
+From a maintainance, performance and security perspective, it is important to keep the footprint of the `gaiad` application as lean as possible.
+
+When adding any new feature, you must ensure that any libraries you wish to include are well maintained and have sufficient usage in the wider ecosystem. This is necessary to avoid having to rework the `gaiad` application at a later date, if a library is longer maintained or is abandoned by its core contributors.
```

Comment:
```suggestion
When adding any new feature, you must ensure that any libraries you wish to include are well maintained and have sufficient usage in the wider ecosystem. This is necessary to avoid having to rework the `gaiad` application at a later date, if a library is no longer maintained or is abandoned by its core contributors.
```

Reply:
That was an important omission, thxs.